doc/api/url.md

```diff
@@ -251,10 +251,10 @@ const myURL = new URL('https://example.org:81/foo');
 console.log(myURL.hostname);
 // Prints example.org
 
-// Setting the hostname does not change the port
+// Setting an invalid hostname is ignored:
 myURL.hostname = 'example.com:82';
 console.log(myURL.href);
-// Prints https://example.com:81/foo
+// Prints https://example.org:81/foo
 
 // Use myURL.host to change the hostname and port
 myURL.host = 'example.org:82';
```

Comment:
I guess we should be changing the hostname here to actually illustrate what the example is about.
```suggestion
myURL.host = 'example.com:82';
```

Reply:
I agree with you. Change hostname and port together using `myURL.host`.